src/main/java/uk/gov/companieshouse/chs/gov/uk/notify/integration/api/mongo/document/NotificationLetterRequest.java

```diff
@@ -1,14 +1,77 @@
 package uk.gov.companieshouse.chs.gov.uk.notify.integration.api.mongo.document;
 
-import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.mongodb.config.EnableMongoAuditing;
 import org.springframework.data.mongodb.core.mapping.Document;
 import org.springframework.data.mongodb.core.mapping.Field;
 import uk.gov.companieshouse.api.chs.notification.model.GovUkLetterDetailsRequest;
 
+import java.time.LocalDateTime;
+
 @Document(collection = "letter_details")
-public record NotificationLetterRequest(
-        @Id String id,
-        @Field("request") GovUkLetterDetailsRequest request
-) {
-    // Empty: using only auto-generated methods
+@EnableMongoAuditing(dateTimeProviderRef = "mongodbDatetimeProvider")
+public class NotificationLetterRequest {
+
+    @Field("created_at") @CreatedDate
+    private LocalDateTime createdAt;
+
+    @Field("updated_at") @LastModifiedDate
+    private LocalDateTime updatedAt;
+
+    private GovUkLetterDetailsRequest request;
```

Comment:
Could we be consistent and use "field" here too? Like the other classes? 
```suggestion
    @Field("request")
    private GovUkLetterDetailsRequest request;
```